precompiles/Modexp.yul

```diff
@@ -11,9 +11,42 @@ object "ModExp" {
             function LIMB_SIZE_IN_BITS() -> limbSize {
                 limbSize := 0x100
             }
-
             // HELPER FUNCTIONS
 
+            /// @notice Stores a one in big unsigned integer form in memory.
+            /// @param nLimbs The number of limbs needed to represent the operand.
+            /// @param toAddress The pointer to the MSB of the destination.
+            function oneWithLimbSizeAt(nLimbs, toAddress) {
+                let pointerToOne :=  toAddress
+                mstore(pointerToOne, 0x1)
+                for { let i := sub(nLimbs, 1) } gt(i, 0) { i := sub(i, 1) } {
+                   let offset := add(mul(i, 32), pointerToOne)
+                   mstore(offset, 0x0)
+                }
+             }
+            
+            /// @notice Stores a zero in big unsigned integer form in memory.
+            /// @param nLimbs The number of limbs needed to represent the operand.
+            /// @param toAddress The pointer to the MSB of the destination.
+            function zeroWithLimbSizeAt(nLimbs, toAddress) {
+                for { let i := 0 } lt(i, nLimbs) { i := add(i, 1) } {
+                    let offset := mul(i, 32)
+                    mstore(add(toAddress, offset), 0)
+                }
+            }
```

Comment:
```suggestion
            function zeroWithLimbSizeAt(nLimbs, toAddress) {
                let overflow := add(toAddress, shl(5, nLimbs))
                for { } lt(toAddress, overflow) { toAddress := add(toAddress, LIMB_SIZE_IN_BYTES()) } {
                    mstore(toAddress, 0)
                }
            }
```

Reply:
I like it, let's use this if the tests run successfully.